otter/grade/containers.py

```diff
@@ -234,4 +245,7 @@ def kill_container():
         os.remove(results_path)
         os.remove(temp_subm_path)
 
-    return df
+    return {
```

Comment:
(Optional) Perhaps a better way of doing this, so we don't need to construct the `df_pts_poss` dataframe every time would be to have the future return `scores` and then write a helper method to turn a list of `GradingResults` objects into a dataframe that the caller can use to convert them. This refactor is optional though -- if you don't have the time then this is fine as-is.